usecase/usecase-wrt-android-tests/tests.android.xml

```diff
@@ -2,7 +2,9 @@
 <?xml-stylesheet type="text/xsl" href="./testcase.xsl"?>
 <test_definition>
   <suite name="Usecase WRT">
-    <set name="Web Runtime" background="brand-primary" icon="glyphicon-asterisk">
+    <set background="brand-primary" icon="glyphicon-asterisk" name="Web Runtime" type="js" ui-auto="bdd">
```

Comment:
here, the execution type and set status need consistent with usecase-webapi, you can confirm details with wanming, and check the desc in wiki: http://otcqa.sh.intel.com/wiki/Crosswalk/Test_Suite_WebDriver_Based_Automation_Management#Usecase_automation_with_bdd